cookbook/frontend/assetic_php.rst

```diff
@@ -0,0 +1,152 @@
+  applications;
+* If you want to simplify application deployment.
+
+In this article you'll learn how to combine and minimize CSS and JavaScript files
+and how to compile Sass SCSS files using pure PHP libraries with Assetic.
+
+Installing the Third-Party Compression Libraries
+------------------------------------------------
+
+Assetic includes a lot of ready-to-use filters but it doesn't include their
+associated libraries. Therefore, before enabling the filters used in this article
+you must install two libraries. Open a command console, browse to your project
+directory and execute the following commands:
+
+.. code-block:: bash
+
+    $ composer require leafo/scssphp
+    $ composer require patchwork/jsqueeze:"~1.0"
+
+It's very important to maintain the ``~1.0`` version constraint for the ``jsqueeze``
+dependency because the most recent stable version is not compatible with Assetic.
+
+Organizing Your Web Asset Files
+-------------------------------
+
+This example shows the very common scenario of using the Bootstrap framework, the
+jQuery library, the FontAwesome icon font and some regular CSS and JavaScript
+application files (called ``main.css`` and ``main.js``). The recommended directory
+structure for this set-up is the following:
+
+.. code-block:: text
+
+    web/assets/
+    ├── css
+    │   ├── main.css
+    │   └── code-highlight.css
+    ├── js
+    │   ├── bootstrap.js
+    │   ├── jquery.js
+    │   └── main.js
+    └── scss
+        ├── bootstrap
+        │   ├── _alerts.scss
+        │   ├── ...
+        │   ├── _variables.scss
+        │   ├── _wells.scss
+        │   └── mixins
+        │       ├── _alerts.scss
+        │       ├── ...
+        │       └── _vendor-prefixes.scss
+        ├── bootstrap.scss
+        ├── font-awesome
+        │   ├── _animated.scss
+        │   ├── ...
+        │   └── _variables.scss
+        └── font-awesome.scss
+
+Combining and Minimizing CSS Files and Compiling SCSS Files
+-----------------------------------------------------------
+
+First, configure a new ``scssphp`` Assetic filter as follows:
+
+.. code-block:: yaml
+
+    # app/config/config.yml
+    assetic:
+        filters:
+            scssphp:
+                formatter: "Leafo\\ScssPhp\\Formatter\\Compressed"
+            # ...
+
+The value of the ``formatter`` option is the fully qualified class name of the
+formatter used by the filter to produce the compiled CSS file. Using the
+compressed formatter allows to minimize the resulting file, no matter if the
+original files are regular CSS files or SCSS files.
+
+Then, update the code of your Twig template to add the ``{% stylesheets %}`` tag
+defined by Assetic:
+
+.. code-block:: html+jinja
+
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <!-- ... -->
+
+            {% stylesheets filter="scssphp" output="css/app.css"
+                "assets/scss/bootstrap.scss"
+                "assets/scss/font-awesome.scss"
```

Comment:
I don't think that's a good idea, because this article is about using PHP only libraries. But once the Bower article written by @WouterJ is published, we can indeed add a link to it in the introduction of this article (where we say "JS is the best  for this"). I think taht would make a lot of sense. Thanks!